apps/journeys-admin/src/components/NewPageWrapper/MainPanelBody/MainPanelBody.tsx

```diff
@@ -30,8 +32,8 @@ export function MainPanelBody({
         sx={{
           // Make optional or remove during cooldown
           // backgroundColor: 'background.paper',
-          px: { xs: 6, sm: 8 },
-          py: { xs: 6, sm: 9 },
+          px: { xs: isEdit != null ? 0 : 6, sm: isEdit != null ? 0 : 8 },
```

Comment:
This needs to be a strict truth check, not a nullish check. If the change above is implemented with a default value we can just do
```suggestion
          px: { xs: disableGutters ? 0 : 6, sm: disableGutters ? 0 : 8 },
```